docs/framework/winforms/controls/treeview-control-windows-forms.md

```diff
@@ -33,8 +33,6 @@ The Windows Forms `TreeView` control displays a hierarchy of nodes, like the way
  [How to: Attach a ShortCut Menu to a TreeView Node](../../../../docs/framework/winforms/controls/how-to-attach-a-shortcut-menu-to-a-treeview-node.md)  
  Demonstrates how to add a shortcut menu to a tree view node.  
   
- Also see [How to: Add and Remove Nodes with the Windows Forms TreeView Control Using the Designer](http://msdn.microsoft.com/library/ms233651\(v=vs.110\)), [How to: Attach a Shortcut Menu to a TreeNode Using the Designer](http://msdn.microsoft.com/library/ms171708\(v=vs.110\)).  
```

Comment:
this exists:
https://docs.microsoft.com/en-us/dotnet/framework/winforms/controls/add-and-remove-nodes-with-wf-treeview-control-using-the-designer

https://docs.microsoft.com/en-us/dotnet/framework/winforms/controls/how-to-attach-a-shortcut-menu-to-a-treenode-using-the-designer

Reply:
fixed thanks!